site/public/js/server.js

```diff
@@ -1750,7 +1750,26 @@ function changePermission(filename, checked) {
 
     $.ajax({
         url: url,
-        success: function(data) {},
+        success: function(data) {csrf_token: csrfToken},
```

Comment:
`fn` is not sent.

Reply:
whoops sorry! I rushed it too much, hopefully this will fix it

Reply:
ok fixed, no errors for me on testing individual files, please let me know if errors persist